Tests/TestVars/FixtureNonSimpleVars.swift

```diff
@@ -1,50 +1,49 @@
 import MockoloFramework
 
-let nonSimpleVars = """
-import Foundation
-
-/// @mockable
-@objc
```

Comment:
This `@objc` annotation has been removed for linux build. That didn't have meanings in this test.